.github/workflows/manual-engine-flutter-roll.yml

```diff
@@ -0,0 +1,61 @@
+on: 
```

Comment:
This is not the right place for this, @keyonghan FYI

Reply:
umm right migrating to cocoon based on the latest code review feedback from https://github.com/flutter/engine/pull/47734#discussion_r1386839129. 

If I am understanding correctly, @CaseyHillers was suggesting that we built a custom action from the code in cocoon, and publish the cocoon code to github marketplace. And then we invoke this self-published marketplace action from the framework/engine repo. 
Or maybe the idea is to have the code live on cocoon for now to trigger fewer presubmit tests?

Reply:
This is probably a better discussion for the design doc. I would recommend focussing on CPs first as those should live in Cocoon since both the engine and framework will consume them.

Reply:
> This is probably a better discussion for the design doc. I would recommend focussing on CPs first as those should live in Cocoon since both the engine and framework will consume them.

umm sure created https://github.com/flutter/cocoon/pull/3246 for manual CPs and sent it out for review.